recipes/mysqlclient/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mysqlclient" %}
+{% set version = "1.3.12" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "2d9ec33de39f4d9c64ad7322ede0521d85829ce36a76f9dd3d6ab76a9c8648e5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - mysql-connector-c
+
+  run:
+    - python
+    - mysql-connector-c
+
+test:
+  imports:
+    - MySQLdb
+    - MySQLdb.constants
+    - MySQLdb.cursors
+
+about:
+  home: https://github.com/PyMySQL/mysqlclient-python
+  doc_url: https://mysqlclient.readthedocs.io/
+  license: GPL 2.0
+  license_file: LICENSE
+  summary: 'Python interface to MySQL'
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```

Comment:
Given some time has passed, are you still interested in helping maintain this, @pmlandwehr?

Reply:
ping @pmlandwehr